File: Conit.DAL/Entities/Instruction.cs
﻿using System;

namespace Conit.DAL.Entities
{
    public class Instruction
    {
        public int Id { get; set; }

        public int ProductId { get; set; }

        public Product Product { get; set; }

        public string Description { get; set; }

        public DateTime DateOfAdding { get; set; }

        public bool IsDeleted { get; set; }
    }
}

File: Mobile/flutter_app/flutter_app/lib/instruction.dart
class Instruction {
  final int id;
  final String description;
  final int productId;

  Instruction._({
    this.id,
    this.description,
    this.productId,
  });

  factory Instruction.fromJson(Map<String, dynamic> json) {
    return new Instruction._(
      id: json['id'],
      description: json['description'],
      productId: json['productId'],
    );
  }
}